tests/scale/scale_test.go

```diff
@@ -0,0 +1,45 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	. "github.com/portworx/torpedo/tests"
+)
+
+func TestBasic(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Torpedo : Scale")
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+})
+
+// This test performs scale test of starting an application and destroying it (along with storage)
+var _ = Describe("Scaled Setup and teardown", func() {
+	It("has to setup, validate and teardown large scale apps", func() {
+		var contexts []*scheduler.Context
+		for i := 0; i < Inst().ScaleFactor; i++ {
+			contexts = ScheduleAndValidate(fmt.Sprintf("scalesetupteardown-%d", i))
```

Comment:
it should be append

Reply:
You mean replace `fmt.Sprintf("scalesetupteardown-%d", i)` with string append? What's wrong with sprintf?

Reply:
Fixed in incremental